src/main/java/uk/ac/ebi/ampt2d/commons/accession/persistence/IAccessionedObjectCustomRepository.java

```diff
@@ -17,11 +17,13 @@
  */
 package uk.ac.ebi.ampt2d.commons.accession.persistence;
 
+import org.springframework.transaction.annotation.Transactional;
+
 import java.util.Collection;
-import java.util.Set;
 
-public interface IAccessionedObjectCustomRepository {
+public interface IAccessionedObjectCustomRepository<ENTITY> {
 
-    <ENTITY> void insert(Collection<ENTITY> entities);
+    @Transactional
```

Comment:
Is it ok to put Transactional here in the custom repository?

Reply:
I see it's on the JPA implementation, but what about other implementations?

Reply:
It will be ignored silently.